exchanges/btse/btse.go

```diff
@@ -329,10 +353,17 @@ func (b *BTSE) GetFee(feeBuilder *exchange.FeeBuilder) (float64, error) {
 	case exchange.CryptocurrencyTradeFee:
 		fee = calculateTradingFee(feeBuilder.IsMaker)
 	case exchange.CryptocurrencyWithdrawalFee:
-		if feeBuilder.Pair.Base.Match(currency.BTC) {
+		switch feeBuilder.Pair.Base {
+		case currency.USDT:
+			fee = 1.08
+		case currency.TUSD:
+			fee = 1.09
+		case currency.BTC:
 			fee = 0.0005
-		} else if feeBuilder.Pair.Base.Match(currency.USDT) {
-			fee = 5
+		case currency.ETH:
+			fee = 0.01
+		case currency.LTC:
+			fee = 0.001
 		}
 	case exchange.InternationalBankDepositFee:
 		fee = getInternationalBankDepositFee(feeBuilder.Amount)
```

Comment:
This fee still isn't accurate:
https://support.btse.com/en/support/solutions/articles/43000064283-what-are-the-btse-trading-fees-
Minimum deposit is now $100, not $1000